.github/dependabot.yaml

```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

Comment:
Please set the timezone `Asia/Tokyo` for all the schedules and the time to `10:00`